docs/blog/_posts/2019-05-24-15th-dotty-milestone-release.md

```diff
@@ -0,0 +1,216 @@
+
+With this release comes a bunch of new features and improvements, such as the ability to enforce whether an operator is intended to be used in an infix position, the type safe pattern bindings and more.
+
+This release serves as a technology preview that demonstrates new
+language features and the compiler supporting them.
+
+Dotty is the project name for technologies that are being considered for
+inclusion in Scala 3. Scala has pioneered the fusion of object-oriented and
+functional programming in a typed setting. Scala 3 will be a big step towards
+realising the full potential of these ideas. Its main objectives are to
+
+- become more opinionated by promoting programming idioms we found to work well,
+- simplify where possible,
+- eliminate inconsistencies and surprising behaviours,
+- build on strong foundations to ensure the design hangs together well,
+- consolidate language constructs to improve the language’s consistency, safety, ergonomics, and
+  performance.
+
+You can learn more about Dotty on our [website](https://dotty.epfl.ch).
+
+<!--more-->
+
+This is our 15th scheduled release according to our
+[6-week release schedule](https://dotty.epfl.ch/docs/contributing/release.html).
+
+# What’s new in the 0.15.0-RC1 technology preview?
+## Operator Rules
+This change addresses the problem of the regulation of whether an operator is supposed to be used in an infix position. The motivation is for the library authors to be able to enforce whether a method or a type is supposed to be used in an infix position by the users. This ability will help to make code bases more consistent in the way the calls to methods are performed.
+
+Methods with symbolic names like `+` are allowed to be used in an infix position by default:
+
+```scala
+scala> case class Foo(x: Int) { def +(other: Foo) = x + other.x }
+// defined case class Foo
+
+scala> Foo(1) + Foo(2)
+val res0: Int = 3
+```
+
+Methods with alphanumeric names are not allowed to be used in an infix position. Breaking this constraint will raise a deprecation warning:
+
+```scala
+scala> case class Foo(x: Int) { def plus(other: Foo) = x + other.x }
+// defined case class Foo
+
+scala> Foo(1) plus Foo(2)
+1 |Foo(1) plus Foo(2)
+  |       ^^^^
+  |Alphanumeric method plus is not declared @infix; it should not be used as infix operator.
+  |The operation can be rewritten automatically to `plus` under -deprecation -rewrite.
+  |Or rewrite to method syntax .plus(...) manually.
+val res1: Int = 3
+
+scala> Foo(1).plus(Foo(2))
+val res2: Int = 3
+```
+
+As the warning says, if you want the users of the method to be able to use it in an infix position, you can do so as follows:
```

Comment:
you can do so -> they can do so

Reply:
"you", since that's the author of the code that has the control over how the method is used. If it is not marked as `infix` by the author, there's nothing (probably) "they" can do to get rid of the warning.